project_report/tex/evaluation.tex

```diff
@@ -1 +1,26 @@
 \section{Evaluation}
+The evaluation section describes how the reflections of the goals, set for the
+project, meets the customer demands, as well as how the development process has
+been.
+
+
+\subsection{Evaluation from the customer side}
```

Comment:
This is validating our solution, not evaluating it. 

Consider writing about:
- Is it user friendly?
  - does it take a long time to understand how to use it etc.
  - Is the design intuiative?
  - How well is the data displayed?

All the good stuff.

Reply:
Maybe we have done it now?